codespeed/views.py

```diff
@@ -288,16 +289,37 @@ def gettimelinedata(request):
 
                 results = []
                 for res in resultquery:
-                    std_dev = ""
-                    if res.std_dev is not None:
-                        std_dev = res.std_dev
-                    results.append(
-                        [
-                            res.revision.date.strftime('%Y/%m/%d %H:%M:%S %z'),
-                            res.value, std_dev,
-                            res.revision.get_short_commitid(), branch
-                        ]
-                    )
+                    if bench.data_type == 'M':
+                        val_min = ""
+                        if res.val_min is not None:
+                            val_min = res.val_min
+                        val_max = ""
+                        if res.val_max is not None:
+                            val_max = res.val_max
+                        q1 = ""
+                        if res.q1 is not None:
+                            q1 = res.q1
+                        q3 = ""
+                        if res.q3 is not None:
+                            q3 = res.q3
+                        results.append(
+                            [
+                                res.revision.date.strftime('%Y/%m/%d %H:%M:%S %z'),
```

Comment:
We could factor out the data formatting somewhere common to both data_types. Even a method on revision `date_formatted()` or similar